### PR TITLE
feat: new promo banner background gradient

### PR DESCRIPTION
**Pull request by vacas5**
Introducing a new oceanBlue version of the background gradient for the
PromoBanner, which has a specific implementation with three stops.

I considered an alternative of allowing this component to take a custom background color.

Closes D2IQ-77630

## Dependencies

Have asked design to give me colors that are already in our design system rather than these three custom colors, so I will wait on them to merge this in, but the implementation can be reviewed.

## Screenshots

Screen Shot 2021-08-09 at 5 05 14 PM

## Checklist for PR creator

- [ ] If any new components were added, there are exported from `packages/index.ts`
- [x] If this PR is associated with a JIRA, it is mentioned in commit message footer ("Closes …")
- [ ] If this PR contains breaking changes, is stated in commit message body ("BREAKING CHANGE: …")
- [ ] Info for applicable sections above is provided

### CODE REVIEW

File: packages/promo/style.ts [hunk 1]
```diff
@@ -42,17 +44,30 @@ export const graphicContainer = css`
...
-      `
-    : "";
+export const getBackgroundGradient = (gradientStyle?: GradientStyle) => {
+  if (!gradientStyle) {
+    return "";
+  }
+
+  const stops = gradientStyles[gradientStyle];
+
+  if (gradientStyle === "oceanBlue") {
```
Comment (clintonmedbery) [inline packages/promo/style.ts:59]: Could we base this on stops.length instead? Not sure how I feel about having oceanBlue just hardcoded here. Not sure it's a huge deal considering these are essentially themes, but that might make this more extensible.

**Review by clintonmedbery — COMMENTED** [review]
Nice work. Just mentioned one issue.

File: packages/promo/style.ts (hunk 1 above)
Comment (mstruebing) [reply packages/promo/style.ts:59]: I'm not sure I agree here, this component can not be generally used as we only provide currently three options.
So if we add one other we probably want some other stops here.
On the other hand, if we would have always the same stops, we could make it very generic and calculate the single stop values from the total amount of the stops.

It's more of a question what we want here, generic component or some specific gradients.

Comment (vacas5) [reply packages/promo/style.ts:59]: the stops for `oceanBlue` can't be mathematically deducted. This is a custom implementation thus, no logic. My original take had some advanced JS trying to calculate the %s based on # of stops but then I saw what design actually asked for.

**Review by mstruebing — APPROVED** [review]
I'm okay with that and as long as we only expose specific gradients it doesn't make a big difference to me.

**Comment by vacas5** [comment]
at some point we might consider allowing this component to accept a custom background, but I think this is the shortest way home right now.

File: packages/promo/style.ts (hunk 1 above)
Comment (clintonmedbery) [reply packages/promo/style.ts:59]: Ok that makes sense.

**Pull request merged**